app/scripts/background.js

```diff
@@ -473,6 +495,10 @@ async function initialize() {
 
     if (!isManifestV3) {
       await loadPhishingWarningPage();
+      const { name } = await browser.runtime.getBrowserInfo();
```

Comment:
According to https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/getBrowserInfo, chrome doesn't support `runtime.getBrowserInfo`, so you'll probably need to find another way to get this information. You might want to look into using `appStateController.store.getState().browserEnvironment.browser`